runtime/tutor/en/vim-01-beginner.tutor.json

```diff
@@ -40,5 +40,11 @@
     "784": "Adding 123 to 456 gives you 579.",
     "810": "a) This is the first item.",
     "811": "b) This is the second item."
+  },
+  "transform": {
+    "win32": {
+      "!ls": "!dir",
+      "!rm": "!del"
```

Comment:
Instead of doing a search/replace on the raw strings, could introduce a format like `{unix:(foo),win:(bar)}` so that the "transform" is given in the tutor contents. 

For example this line: https://github.com/neovim/neovim/blob/ca825f38732252bfe8d285c15fcf1b3d45f2c119/runtime/tutor/en/vim-01-beginner.tutor#L632

would become:

     1. Type `:!{unix:(ls),win:(dir)}`{vim} to get a listing of your directory.

That should also greatly simplify the `tutor#ApplyTransform()` logic. It would just be a single `:%s` call:

```
if has('win32')
  %s/\v\{unix:\(([^)]+)\).*win:\(([^)]+)\)}/\2
else
  %s/\v\{unix:\(([^)]+)\).*win:\(([^)]+)\)}/\1
endif